### PR TITLE
[WIP] Responsibilties of working group leads

### PR DESCRIPTION
**Pull request by dasanchez**
Updated governance doc with the tasks that each working group lead is responsible for.

### CODE REVIEW

**Comment by Shrinks99** [comment]
My name is in there as "Hank" and no my GitHub username, should is be the GitHub username?  I only ask because there doesn't seem to be much consistency in the naming of people and maybe there should be.

**Comment by dasanchez** [comment]
@Shrinks99 it's your Matrix username ;)
Would you like me to switch all instances of Hank to Shrinks99?

**Comment by Shrinks99** [comment]
Naw, I just didn't realize it was the Matrix username.  Works for me!  Approved

**Review by Shrinks99 — APPROVED** [review]

File: governance/coordination-structure.md [hunk 1]
```diff
@@ -26,107 +23,136 @@ Toronto Mesh uses voting to determine whether we have reached consensus:
 
 ## Working Groups
 
+### Responsibilities of Working Group Leads
+
+Central Org
+  - Organize meetups and planning sessions
+  - Post event details to the tomesh.net website 
+
+Tools
```
Comment (benhylau) [inline governance/coordination-structure.md:32]: There are more! All of these: https://github.com/tomeshnet/documents/tree/master/service_setup

File: governance/coordination-structure.md [hunk 2]
```diff
@@ -26,107 +23,136 @@ Toronto Mesh uses voting to determine whether we have reached consensus:
...
+  - Maintain the tomesh.net website server 
+  - Maintain the Matrix server
+
+Deployment
+  - Coordinate node deployment efforts
+
+Website
+  - Maintain the tomesh.net website
+    - Update whenever security issues are found
```
Comment (benhylau) [inline governance/coordination-structure.md]: Probably don't need to list this, it's pretty minor, and even if we don't update security patches it's a static site and pretty safe

**Comment by benhylau** [comment]
Thanks for doing this @dasanchez I added my pretty minor comments

**Review by benhylau — APPROVED** [review]

File: governance/coordination-structure.md (hunk 1 above)
Comment (dasanchez) [reply governance/coordination-structure.md:32]: Nice, thanks for the link!

**Pull request merged**